src/lib.rs

```diff
@@ -1,7 +1,10 @@
 extern crate arbitrary;
+extern crate strum;
+#[macro_use]
+extern crate strum_macros;
 #[deny(warnings)]
 #[deny(bad_style)]
-#[deny(missing_docs)]
```

Comment:
I'm not super enthusiastic about allowing missing docs. Is this because of an interaction with the strum generated code?

Reply:
Sorry looks like I forgot to mention this when submitting.  This needs fixing upstream: the generated count const is undocumented.  It's probably an easy patch to land as they landed one for EnumIter.

Reply:
PR'd: https://github.com/Peternator7/strum/pull/43